### PR TITLE
Add recipe for sphinx_gmt

### PR DESCRIPTION
**Pull request by leouieda**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by leouieda** [comment]
@seisman I included you as a maintainer, is that OK with you?

**Comment by seisman** [comment]
OK.

**Comment by leouieda** [comment]
@conda-forge/help-python this seems good to merge. Could you please take a look when you have some time?

File: recipes/sphinx_gmt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 99889d655237928bf6c3d4d1b7abef2e8252342a0cf203f2f88eccef2db13a62
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```
Comment (mwcraig) [inline recipes/sphinx_gmt/meta.yaml]: I think this recipe could be noarch if you remove this selector (comment below on how to modify to include your version restriction)

File: recipes/sphinx_gmt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - python
```
Comment (mwcraig) [inline recipes/sphinx_gmt/meta.yaml]: Add your python constraint here

```suggestion
    - python >=3.5
```

File: recipes/sphinx_gmt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - pip
+  run:
+    - python
```
Comment (mwcraig) [inline recipes/sphinx_gmt/meta.yaml]: and here

```suggestion
    - python >=3.5
```

**Comment by lebedov** [comment]
@mwcraig I think you meant @leouida.

**Comment by mwcraig** [comment]
Indeed 🤦‍♂

**Comment by leouieda** [comment]
Thanks @mwcraig, don't know how I missed that. Done and builds passing :+1:

**Comment by mwcraig** [comment]
thanks, merging!

**Pull request merged**